vdoctl/cmd/root.go

```diff
@@ -69,6 +70,8 @@ vdoctl configure vc
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	cobra.CheckErr(rootCmd.Execute())
+	GenerateMarkdownDoc()
```

Comment:
IS this something which will keep getting generated everytime someone runs the vdoctl command, I think this will be an issue if some user just downloads our `vdoctl` and executes, the docs will get auto-populated even if user doesn't wants it.
..is my understanding correct on this behaviour?